### ISSUE TITLE
Using sed in 'script/dotfiles/create' does not work on linux

### ISSUE THREAD

**Issue by Inmovilizame** [CodelyTV/dotly#16]
Due to different behaviours in `sed` for MacOS (BSD) and Linux (GNU) there is no way to make an in-place substitution that works on both OS families.

```shell
# MacOS 
sed -i '' -e ... 

# Linux
sed -i -e ...
```

The only way I've found to make it work on both specifications was to use [this workaround](https://stackoverflow.com/questions/5694228/sed-in-place-flag-that-works-both-on-mac-bsd-and-linux):

```shell
# Create in both OS families a '.bak' file that can be deleted
sed -i.'bak' -e ...

# Create in both OS families a '.' file that can be deleted
sed -i.'' -e ...
```

I had to face this issue while making some common Makefile used on linux and macos.

PD: Is it ok to contribute? Might be an option to use the os detection that already being used in dotly to apply a proper sed in each case... I know is dirty, but the only solution available so far if using sed is mandatory :)

PD2: Super happy with my CodelyTV susbcription and working on my dotfiles!!

**Comment by Inmovilizame** [comment]
Ok!

Maybe another workaround could through the installer of dotly, by installing with both a new version of bash and gnu-sed. 

```
brew install bash brew install gnu-sed
```

PD: I got it from the bash course :P

**Comment by EduardoSimon** [comment]
I tried your workaround in Ubuntu whitout any luck. Im getting the following error:

**Comment by rgomezcasas** [comment]
This should be fixed with #26 😊

Thanks for reporting!!
